cmd/vclusterctl/cmd/connect.go

```diff
@@ -224,6 +226,13 @@ func (cmd *ConnectCmd) Connect(vclusterName string, command []string) error {
 		return fmt.Errorf("unexpected kube config")
 	}
 
+	if cmd.KubeConfigContextName != "Default" {
+		ctx := kubeConfig.Contexts["Default"]
```

Comment:
Also make sure to rename the cluster as well as auth config to the new name so that merging a kube config doesn't yield any conflicts in clusters or auths